css/_welcome_page.scss

```diff
@@ -4,90 +4,70 @@ body.welcome-page {
 }
 
 .welcome {
+    background-color: $welcomePageHeaderBackground;
+    display: flex;
+    flex-direction: column;
     font-family: $welcomePageFontFamily;
-    height: 100%;
+    justify-content: space-between;
+    min-height: 100vh;
     position: relative;
 
     .header {
         align-items: center;
-        background: $welcomePageHeaderBackground;
         display: flex;
         flex-direction: column;
+        min-height: fit-content;
         overflow: hidden;
         position: relative;
         text-align: center;
 
         .header-text {
             display: flex;
             flex-direction: column;
-            justify-content: space-around;
-            margin-top: 120px;
-            margin-bottom: 20px;
+            margin-top: 156px;
+            margin-bottom: 36px;
             max-width: calc(100% - 40px);
-            min-height: 286px;
-            width: 645px;
+            width: 800px;
+            z-index: $zindex2;
         }
 
         .header-text-title {
             color: $welcomePageTitleColor;
-            font-size: 48px;
+            font-size: 40px;
+            font-weight: 700;
             letter-spacing: -1px;
-            line-height: 58px;
-            margin-bottom: 20px;
+            line-height: 48px;
+            margin-bottom: 16px;
         }
 
         .header-text-description {
             color: $welcomePageDescriptionColor;
             font-size: 20px;
-            line-height: 28px;
-            opacity: 0.8;
-        }
-
```

Comment:
I believe the following fit better:    
    font-size: 1rem;
    line-height: 24px;
    font-weight: 400;